app/components/common/Navigation.vue

```diff
@@ -0,0 +1,534 @@
+<script>
+  const CODECOMBAT = 'codecombat'
+  const CODECOMBAT_CHINA = 'koudashijie'
+  const OZARIA = 'ozaria'
+  const OZARIA_CHINA = 'aojiarui'
+
+  // Use in local development to inform which repo you are in
+  const LOCALHOST_FAKE_HOST = () => application.isProduction() ? null : CODECOMBAT // 'codecombat' | 'ozaria'
+
+  /**
+   * Unified navigation bar component between CodeCombat and Ozaria.
+   * This must be copied exactly to the Ozaria repo.
+   */
+  export default Vue.extend({
+    computed: {
+      isOldBrowser () {
+        if (window.features.china && $.browser) {
+          if (!($.browser.webkit || $.browser.mozilla || $.browser.msedge)) {
+            return true
+          }
+          const majorVersion = $.browser.versionNumber
+          if ($.browser.mozilla && majorVersion < 25) {
+            return true
+          }
+          if ($.browser.chrome && majorVersion < 72) { // forbid some chinese browser
+            return true
+          }
+          if ($.browser.safari && majorVersion < 6) { // 6 might have problems with Aether, or maybe just old minors of 6: https://errorception.com/projects/51a79585ee207206390002a2/errors/547a202e1ead63ba4e4ac9fd
+            return true
+          }
+        }
+        return false
+      },
+
+      isCodeCombat () {
```

Comment:
Seems like these product-determining functions should be accessible throughout our codebase, and not necessarily just in this particular Vue Component, right?

Reply:
Yeah. I can also move these into a utils file. Great callout!

Reply:
I initially kept these scoped as I was keeping it simple to copy paste the logic between the two codebases. But it's much better to isolate these into the utils file now that it works.